recipes/civetweb/all/conanfile.py

```diff
@@ -0,0 +1,107 @@
+    }
+    default_options = {
+        "shared"            : False,
+        "fPIC"              : True,
+        "enable_ssl"        : True,
+        "enable_websockets" : True,
+        "enable_ipv6"       : True,
+        "enable_cxx"        : True
+    }
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == 'Windows':
+            del self.options.fPIC
+
+    def configure(self):
+        if not self.options.enable_cxx:
+            del self.settings.compiler.libcxx
+
+    def requirements(self):
+        if self.options.enable_ssl:
+            self.requires("openssl/1.1.1i")
+
+    def _configure_cmake(self):
+        cmake = CMake(self)
+        cmake.verbose = True
+        cmake.definitions["CIVETWEB_ENABLE_SSL"] = self.options.enable_ssl
+        cmake.definitions["CIVETWEB_ENABLE_WEBSOCKETS"] = self.options.enable_websockets
+        cmake.definitions["CIVETWEB_ENABLE_IPV6"] = self.options.enable_ipv6
+        cmake.definitions["CIVETWEB_ENABLE_CXX"] = self.options.enable_cxx
+        cmake.definitions["CIVETWEB_BUILD_TESTING"] = False
+        cmake.definitions["CIVETWEB_ENABLE_ASAN"] = False
+        cmake.configure(
+                source_dir=os.path.relpath(self._source_subfolder, self._build_subfolder),
+                build_dir=self._build_subfolder)
+        return cmake
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("civetweb-%s" % self.version, self._source_subfolder)
+        tools.replace_in_file(file_path=os.path.join(self._source_subfolder, "CMakeLists.txt"),
+                              search="project (civetweb)",
+                              replace="""project (civetweb)
+                                 include(../conanbuildinfo.cmake)
+                                 conan_basic_setup()""")
+
+    def build(self):
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        self.copy(os.path.join(self._source_subfolder, "LICENSE.md"), dst="licenses")
+        cmake = self._configure_cmake()
+        cmake.install()
+        shutil.rmtree(os.path.join(self.package_folder, "lib", "cmake"))
+        bin_folder = os.path.join(self.package_folder, "bin");
+        for bin_file in os.listdir(bin_folder):
+            if not bin_file.startswith("civetweb"):
+                os.remove(os.path.join(bin_folder, bin_file))
+
+    def package_info(self):
+        self.cpp_info.libs = tools.collect_libs(self)
+        if self.settings.os == "Linux":
+            self.cpp_info.libs.extend(["dl", "rt", "pthread"])
+            if self.options.enable_cxx:
+                self.cpp_info.libs.append("m")
+        elif self.settings.os == "Macos":
+            self.cpp_info.exelinkflags.append("-framework Cocoa")
```

Comment:
there's a frameworks and/or system libs that exists for mac... there's example in the repo

Reply:
```suggestion
            self.cpp_info.frameworks.append("Cocoa")
```